SimTracker/SiPixelDigitizer/test/PixelDigisTest.cc

```diff
@@ -103,7 +103,7 @@ using namespace std;
 // class decleration
 //
 
-class PixelDigisTest : public edm::EDAnalyzer {
+class PixelDigisTest : public edm::one::EDAnalyzer<> {
```

Comment:
This class appears to need
```suggestion
class PixelDigisTest : public edm::one::EDAnalyzer<edm::one::SharedResources> {
```
and
```c++
  usesResource(TFileService::kSharedResource);
```
in the constructor.